py/__init__.py

```diff
@@ -23,6 +23,15 @@ def py_get_import_path_holders(param_type):
     return import_paths.get(param_type, [])
 
 
+_private_custom_types = ("SqlError", "SqlQueryId")
+
+
+def py_custom_type_name(name):
+    if name in _private_custom_types:
+        return "_" + name
+    return name
```

Comment:
Did not we need this one before? Is this the first time a private custom type is introduced?

Reply:
Yes, this is the first time